Cannot resolve dependecy

I can't pull in dependency when using sbt or gradle

Full sbt build file

```
enablePlugins(JavaAppPackaging)

name := "sample"
version := "1.0"
scalaVersion := "2.12.3"

scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8")

libraryDependencies ++= {
  val scalaTestVersion  = "3.0.1"
  Seq(
    "org.scalaz" %% "scalaz-zio" % "0.1-SNAPSHOT",

    //Logging
    "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
    "ch.qos.logback" % "logback-classic" % "1.2.3",

    //Testing
    "org.scalatest"     %% "scalatest" % scalaTestVersion % "test"
  )
}

mainClass in Compile := Some("Rest")

Revolver.settings
```

Results in the following error

```
[info] Resolving org.scalaz#scalaz-zio_2.12;0.1-SNAPSHOT ...
[info] Resolving org.scalaz#scalaz-zio_2.12;0.1-SNAPSHOT ...
[warn] 	module not found: org.scalaz#scalaz-zio_2.12;0.1-SNAPSHOT
[warn] ==== local: tried
[warn]   /Users/tom/.ivy2/local/org.scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/scalaz-zio_2.12-0.1-SNAPSHOT.pom
[warn] ==== local-preloaded-ivy: tried
[warn]   /Users/tom/.sbt/preloaded/org.scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/ivys/ivy.xml
[warn] ==== local-preloaded: tried
[warn]   file:////Users/tom/.sbt/preloaded/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/scalaz-zio_2.12-0.1-SNAPSHOT.pom
[info] Resolving com.typesafe.scala-logging#scala-logging_2.12;3.9.0 ...
```

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scalaz#scalaz-zio_2.12;0.1-SNAPSHOT: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.scalaz:scalaz-zio_2.12:0.1-SNAPSHOT (/Users/tom/repos/akka-http-microservice/build.sbt#L9-22)
[warn] 		  +- sample:sample_2.12:1.0
[trace] Stack trace suppressed: run 'last *:ssExtractDependencies' for the full output.
[trace] Stack trace suppressed: run 'last *:update' for the full output.
[error] (*:ssExtractDependencies) sbt.ResolveException: unresolved dependency: org.scalaz#scalaz-zio_2.12;0.1-SNAPSHOT: not found

[error] (*:update) sbt.ResolveException: unresolved dependency: org.scalaz#scalaz-zio_2.12;0.1-SNAPSHOT: not found
[error] Total time: 1 s, completed 02-Aug-2018 21:06:26
```

Also tried in gradle

```
compile "org.scalaz:scalaz-zio_${scalaVersion}:0.1-SNAPSHOT"
```

Results in the following error

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compileClasspath'.
> Could not find org.scalaz:scalaz-zio_2.12:0.1-SNAPSHOT.
  Searched in the following locations:
      https://jcenter.bintray.com/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/maven-metadata.xml
      https://jcenter.bintray.com/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/scalaz-zio_2.12-0.1-SNAPSHOT.pom
      https://jcenter.bintray.com/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/scalaz-zio_2.12-0.1-SNAPSHOT.jar
      https://repo1.maven.org/maven2/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/maven-metadata.xml
      https://repo1.maven.org/maven2/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/scalaz-zio_2.12-0.1-SNAPSHOT.pom
      https://repo1.maven.org/maven2/org/scalaz/scalaz-zio_2.12/0.1-SNAPSHOT/scalaz-zio_2.12-0.1-SNAPSHOT.jar
```

Also, cannot find scalaz-zio on maven central by searching for it.

--- a/README.md
+++ b/README.md
@@ -31,10 +31,12 @@ Use `IO` because it's simply not practical to write real-world, correct software
 
 The current snapshot release of ZIO is 0.1, which is cross-built against Scala 2.11.x and 2.12.x.
 
-If you're using SBT, add the following line to your build file:
+If you're using SBT, add the following lines to your build file:
 
 ```scala
 libraryDependencies += "org.scalaz" %% "scalaz-zio" % "0.1-SNAPSHOT"
+
+resolvers += Resolver.sonatypeRepo("snapshots")
 ```
 For Maven and other build tools, you can visit <https://search.maven.org> and search for Scalaz.
 

